wagtail_wordpress_import/prefilters/constants.py

```diff
@@ -0,0 +1,297 @@
+# *****************************
```

Comment:
I wonder if, now that you've split each filter into separate models, it makes sense to merge the constants into the relevant filter module to co-locate the code better (this can make it easier for an unfamiliar developer to navigate the code)